e2e/utils_test.go

```diff
@@ -191,7 +192,7 @@ func setupCluster(configFiles ...string) ([]*gdProcess, error) {
 	}
 
 	// restclient instance that will be used for peer operations
-	client := initRestclient(gds[0].ClientAddress)
+	client := initRestclient(gds[0].ClientAddress, gds[0].LocalStateDir)
```

Comment:
As a cleaner way, you could've passed `*gdProcess` to `initRestclient`.

Reply:
as this is merged I will send a patch to pass ```*gdProcess``` to ```initRestclient```